st2client/st2client/commands/pack.py

```diff
@@ -200,11 +200,16 @@ def run(self, args, **kwargs):
 
     def _get_content_counts_for_pack(self, args, **kwargs):
```

Comment:
This function needs a test, but I don't blame you for tackling that in this PR given your changes.